File: 백준/Bronze/2083. 럭비 클럽/럭비 클럽.java
import java.util.*;

class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while(true) {
            String str = sc.nextLine();
            if(str.equals("# 0 0"))
                break;
            StringTokenizer st = new StringTokenizer(str, " ");
            String name = st.nextToken();
            int age = Integer.parseInt(st.nextToken());
            int weight = Integer.parseInt(st.nextToken());
            if(age > 17 || weight >= 80) {
                System.out.println(name + " Senior");
            } else {
                System.out.println(name + " Junior");
            }
        }
        sc.close();
    }
}

File: 백준/Bronze/2083. 럭비 클럽/럭비 클럽.kt
fun main() {
    while (true) {
        val input = readln()
        if (input == "# 0 0") return
        val tokens = input.split(" ")
        val name = tokens[0]
        val age = tokens[1].toInt()
        val weight = tokens[2].toInt()
        println("$name ${if (age > 17 || weight >= 80) "Senior" else "Junior"}")
    }
}